### DISCUSSION TITLE
Agent会自动修改"/etc/sysconfig/network"文件吗

### DISCUSSION THREAD

**Discussion by zhimibuwudn** [nezhahq/nezhahq.github.io#45] [Q&A] [answered]
几台服务器，在两台机器上安装了Agent，结果当天正常，第二天这两台机器验证机器名称就异常了，发现是/etc/sysconfig/networ   中 HOSTNAME被删除了

**Comment by naiba** [comment, marked as answer]
**绝对不会**，agent 除了自动更新不会自动做任何操作除非你自己执行计划任务。

**Reply by zhimibuwudn** [reply]
ok
